scopes/harmony/application/run.cmd.ts

```diff
@@ -30,6 +31,7 @@ export class RunCmd implements Command {
     ['v', 'verbose', 'show verbose output for inspection and print stack trace'],
     // ['', 'skip-watch', 'avoid running the watch process that compiles components in the background'],
     ['w', 'watch', 'watch and compile your components upon changes'],
+    ['a', 'args [argv]', 'the arguments passing to the app.'],
```

Comment:
@Jinjiang , I guess `--args` flag must be provided with values. If so, change it from `'args [argv]'` to `'args <argv>'`. 
Also, if it's possible to accept multiple args maybe describe how to enter them (e.g. wrap with quotes and separated by commas).

Reply:
Agree. I've updated the description. Feel free to adjust it to be better. 🙏